pkg/ebpf/c/tracee.bpf.c

```diff
@@ -288,7 +288,49 @@ Copyright (C) Aqua Security inc.
 #define get_kconfig(x) CONFIG_##x
 #endif
 
+#ifdef CORE
+
 #define ARCH_HAS_SYSCALL_WRAPPER        1000U
+#define FLATMEM                         1001U
+#define	DISCONTMEM                      1002U
+#define	SPARSEMEM                       1003U
+#define	SPARSEMEM_VMEMMAP               1004U
+#define	SPARSEMEM_EXTREME               1005U
+#define	DYNAMIC_MEMORY_LAYOUT           1006U
+#define	X86_32                          1007U
+#define	X86_PAE                         1008U
```

Comment:
So these values are the keys for the map initialized in the user-mode in the `initialization/kconfig.go` file.

They are defined only for CORE because in no-CORE we have all defines in the header files we import (the `#include <linux/kconfig.h>` we do). So, to use them we just concat the name to `CONFIG_` and we have the values.
This mechanism was established before this PR.